src/validate_oNHM_maps.R

```diff
@@ -0,0 +1,38 @@
+library(sf)
+library(ggplot2)
+
+validate_oNHM_maps <- function(bad_data_hruids, mapfilename, var, title) {
+  
+  # Read in the shapefiles for hrus and background states layer so we can map the hrus that are flagged
+  hrus <- read_sf(dsn=".",layer="hrus")
+  states <- read_sf(dsn=".",layer="states")
```

Comment:
I usually use `st_read`, but it looks like `read_sf` is an alias, so I learned something new!

The `dsn="."` is confusing to me because it is not clear to me what file is being read.

Reply:
hmm ok, i just mean for it to be the current directory/folder, and the layer name there layer= is the name of the shapefile also in the same current directory/folder.

Reply:
huh, that is interesting. So you have two shapefiles in that folder, `hrus.shp` and `states.shp`. I normally do this:

```
hrus <- st_read("hrus.shp")
states <- st_read("states.shp")
```

but no worries if it works! Just trying to understand a syntax I hadn't seen.